src/extremeweatherbench/calc.py

```diff
@@ -0,0 +1,594 @@
+import dataclasses
+from collections import namedtuple
+from itertools import product
+from typing import Optional, Sequence
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import xarray as xr
+from scipy import spatial
+from skimage import measure
+from skimage.feature import peak_local_max
+
+from extremeweatherbench import inputs
+
+Location = namedtuple("Location", ["latitude", "longitude"])
+
+
+# TODO: determine if this is needed or can be removed for the refactor
+@dataclasses.dataclass
+class TC:
+    id: int
+    valid_time: pd.Timestamp
+    coordinate: Location
+    vmax: float
+    slp: float
+
+
+# TODO: determine if this is needed or can be removed for the refactor
+@dataclasses.dataclass
+class TCTracks:
```

Comment:
This is just a single track though, isn't it? The plural naming is confusing. Is it tracks from multiple cyclones or the track timestamps of a single cyclone, or something else?